Model/Rate/Result.php

```diff
@@ -71,14 +71,15 @@ class Result extends \Magento\Shipping\Model\Rate\Result
 	 */
     public function __construct(
         \Magento\Store\Model\StoreManagerInterface $storeManager,
-        \Magento\Backend\Model\Session\Quote $quote,
+        //\Magento\Backend\Model\Session\Quote $quote,
+        \Magento\Checkout\Model\Session $checkoutSession,
 	    Session $session,
         Checkout $myParcelHelper,
         PackageRepository $package
     ) {
         parent::__construct($storeManager);
 
-
+        $this->products = $checkoutSession->getQuote()->getItems();
```

Comment:
Ook op regel 89 staat `$this->product`. Het kan zijn dat die regel weg moet.
https://github.com/myparcelnl/magento/blob/fee7db642749402bf48dc32c3cb357428c442990/Model/Rate/Result.php#L89